pyproject.toml

```
@@ -27,7 +27,7 @@ slack-sdk = "^3.4.2"
 cryptography = "^3.4.7"
 
 [tool.poetry.dev-dependencies]
-black = "*"
+black = "^20.8b1"
```

```suggestion
black = "*"
```

Leaving at * as the poetry lock file handles the versioning.

---

That's the problem I had... running Poetry lock did not handle the "*".

---

nautobot-plugin-chatops % poetry lock
The lock file might not be compatible with the current version of Poetry.
Upgrade Poetry to ensure the lock file is read properly or, alternatively, regenerate the lock file with the `poetry lock` command.
Updating dependencies
Resolving dependencies... (0.8s)
[SolverProblemError]
Because nautobot-chatops depends on black (*) which doesn't match any versions, version solving failed.

Same error when running `Poetry update`.

---

Can you check what version of Poetry you have?

---

I thought I had an updated version, but turns out I don't. I will try this again.